### ISSUE TITLE
I have a question about model size,the lstm model which i trained is about 78.6MB.It's two times bigger than yours(only 30.7MB). What's the matter？

### ISSUE THREAD

**Issue by Courageux-J** [jiangsutx/SRN-Deblur#20]

**Comment by jiangsutx** [comment]
Do you mean the saved model from TensorFlow ?
I think you saved intermediate values, such as gradients for each variable.

In our released model, only parameters used in inference stage are saved.

**Comment by Courageux-J** [comment]
thanks for your reply!
i want to train new model by tuning parameters with your released lstm model 
how can i continue training with your model?
and another question,how can i make the model size as  the same as your released model size?
thank you very much~

**Comment by noranali** [comment]
> thanks for your reply! i want to train new model by tuning parameters with your released lstm model how can i continue training with your model? and another question,how can i make the model size as the same as your released model size? thank you very much~

hello , could you help me and tell me how to compute the trainable parameters please?

**Comment by jiangsutx** [comment]
> > thanks for your reply! i want to train new model by tuning parameters with your released lstm model how can i continue training with your model? and another question,how can i make the model size as the same as your released model size? thank you very much~
> 
> hello , could you help me and tell me how to compute the trainable parameters please?

Sorry that this version of Tensorflow is very old. You may search Tensorflow + flops to see if there are available tools.
Actually the model only use conv layers and LSTM, the layers and parameters defined in the code [model.py](https://github.com/jiangsutx/SRN-Deblur/blob/master/models/model.py), it would not be very difficult to calculate the value.

If you don't want to manually calculate, you may also write into more recent Pytorch and calculate params. It woundn't be very complicated. 

Sorry again for such inconvenience.

**Comment by noranali** [comment]
Dear Professor Xin Tao
thanks very much for you replay but can you help me to find the online demo or the code of the paper "Learning a Convolutional Neural Network for Non-uniform Motion Blur Removal ".
________________________________
